common/lib/xmodule/xmodule/course_module.py

```diff
@@ -855,6 +855,14 @@ class CourseFields(object):
         scope=Scope.settings
     )
 
+    minimum_grade_credit = Float(
```

Comment:
@wedaly Thanks for the note-- please do check with Griff, but if he is OK with it, it's fine with me.